firebase-firestore/src/androidTest/java/com/google/firebase/firestore/testutil/IntegrationTestUtil.java

```diff
@@ -125,6 +128,10 @@ public enum TargetBackend {
 
   private static boolean backendPrimed = false;
 
+  private static final Random RANDOM = new Random();
```

Comment:
nit: I think it'd be better to move this into the `getLargestDocContent()` function.

```java
final Random random = new Random();
random.nextBytes(bytes);
```